load-code/pe/enumerate-pe-sections.yml

```diff
@@ -5,7 +5,7 @@ rule:
     author: "@Ana06"
     scope: function
     mbc:
-     - Memory::{Micro-objective}::{Method} [C00xx]
+      - Data::Code::Enumerate PE Sections [C0062.001]
```

Comment:
Instead of Code being a micro-behavior under Data, how about defining Discovery::Code Discovery::Enumerate PE Sections [B0046.001]